src/store/app.ts

```diff
@@ -11,16 +9,8 @@ export type BrandInfo = DisplayInfo<'nat'> & {
   petname: string;
 };
 
-export const brandToInfoAtom = mapAtom<Brand, BrandInfo>();
-
-export const walletAtom = atom<ERef<any>>(null);
-
-export const offersAtom = atom<Array<any> | null>(null);
-
-export const pursesAtom = atom<Array<PursesJSONState> | null>(null);
-
-/** A map of anchor brand petnames to their instance ids. */
-export const instanceIdsAtom = mapAtom<string, string>();
+// XXX never filled so all the display functions are generic
+const brandToInfoAtom = mapAtom<Brand, BrandInfo>();
```

Comment:
So any code that uses display functions is broken? We should rip out display functions and all that code to be safe

Reply:
Not broken, IIUC. They just operate without the brand info map. e.g. getDecimalPlaces reads from `wellKnownBrands`. `getPetname` is always empty string.

There is more tech debt to burn down here but that needn't block the improvements made by this PR. This PR as is cuts a lot of dead code and lights the way to more tech debt that can be removed.